packages/mobile/src/components/now-playing-drawer/TrackingBar.tsx

```diff
@@ -59,7 +59,10 @@ export const TrackingBar = (props: TrackingBarProps) => {
     currentAnimation.current = Animated.timing(translateXAnimation.current, {
       toValue: 1,
       duration: timeRemaining * 1000,
-      useNativeDriver: true
+      // Can't use native driver because this animation is potentially hours long,
+      // and would have to be serialized into an array to be passed to the native layer.
+      // The array exceeds the number of properties allowed in hermes
+      useNativeDriver: false
```

Comment:
damn this is WILD. maybe we can do something based on track length? regardless. holy hell great work.

Reply:
Maybe but I think serializing even a 3 minute animation is potentially expensive. I wonder if reanimated would have the same problem